server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -790,6 +791,7 @@ protected Node(
 
             final SearchRequestStats searchRequestStats = new SearchRequestStats(clusterService.getClusterSettings());
             final SearchRequestSlowLog searchRequestSlowLog = new SearchRequestSlowLog(clusterService);
+            final SearchRequestCoordinatorTrace searchRequestCoordinatorTrace = new SearchRequestCoordinatorTrace(tracer);
```

Comment:
With https://github.com/opensearch-project/OpenSearch/pull/12103/files#r1478875661, it will be covered by same traceable listener